### DISCUSSION TITLE
websocket request was bolcked when proxied by nginx

### DISCUSSION THREAD

**Discussion by Buringcarbon** [outline/outline#7332] [Self Hosting]
nginx config:
  location / {
    proxy_pass http://outline:3000/;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "Upgrade";
    proxy_set_header Host $host;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;proxy_set_header Host $host;
    proxy_set_header Host $http_host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Scheme $scheme;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_redirect off;
  }
}

errors in browser console：
MultiplayerEditor.tsx:67  Mixed Content: The page at 'https://wiki.test.com/doc/centosdocker-ufTstYxL9E' was loaded over HTTPS, but attempted to connect to the insecure WebSocket endpoint 'ws://wiki.test.com/collaboration/document.6ea08a25-e437-4dbd-8777-f0e2fd6d6847'. This request has been blocked; this endpoint must be available over WSS.

**Comment by tommoor** [comment]
Most likely `URL` env does not start with "https"

**Reply by Buringcarbon** [reply]
Thanks a lot！！！！, your support has fixed my issue
